examples/util/modelZoo.js

```diff
@@ -1,6 +1,7 @@
 const modelZoo = {
   imageClassificationModels: [{
     modelName: 'MobileNet v1 (TFLite)',
+    framework: ['WebNN'],
```

Comment:
@akineeic Thanks for adding ‘framework’ field to all models description, last I default set framework as 'WebNN', so I didn't add all of them.

Reply:
Because the add frozen tensorflow models  only work with OpenCV.js, so I add the framework to distinguish them. I'm glad that works. Thanks for your review!